Gemfile

```diff
@@ -1,27 +1,29 @@
 source 'https://rubygems.org'
 
-gem 'rails', '4.2.11.1'
+ruby '2.7.1'
+gem 'rails', '~> 6.0.3', '>= 6.0.3.4'
 gem 'pg'
-gem 'sass-rails', '~> 5.0.2'
-gem 'uglifier', '>= 1.3.0'
-gem 'coffee-rails', '~> 4.1.0'
+gem 'puma', '~> 4.1'
+gem 'sass-rails', '>= 6'
+gem 'webpacker', '~> 4.0'
+gem 'uglifier'#, '>= 1.3.0'
+gem 'coffee-rails'#, '~> 4.1.0'
 gem 'haml-rails'
 gem 'jquery-rails'
-gem 'jquery-ui-rails'
+gem 'jquery-ui-rails', '5.0.3'
 gem 'turbolinks'
-gem 'jbuilder', '~> 2.0'
-gem 'delayed_job', '~> 4.0.6'
-gem 'delayed_job_active_record', '~> 4.0.3'
-gem "highcharts-rails", "~> 3.0.0"
+gem 'jbuilder'#, '~> 2.0'
+gem 'delayed_job'#, '~> 4.0.6'
+gem 'delayed_job_active_record'#, '~> 4.0.3'
+gem "highcharts-rails"#, "~> 3.0.0"
 gem 'd3_rails'
 gem 'c3-rails'
 gem 'faye-authentication', github: 'branch14/faye-authentication'
 gem 'devise'
-gem 'sprockets', '~> 3.0'
+gem 'sprockets'#, '~> 3.0'
 gem 'mime-types'
-gem 'sdoc', '~> 0.4.0', group: :doc
-gem 'sqlite3', '~> 1.3.6'
-
+gem 'sdoc'#, '~> 0.4.0', group: :doc
+gem 'sqlite3'#, '~> 1.3.6'
```

Comment:
Please pin the versions of the dependencies to the ones which are currently installed. Unpinned dependencies will break application as soon as there is a breaking change 

And we don't have any CI build to check if something is breaking or not